salt/modules/system.py

```diff
@@ -12,7 +12,8 @@ def __virtual__():
     Only supported on POSIX-like systems
     '''
     if salt.utils.is_windows() or not salt.utils.which('shutdown'):
-        return False
+        return (False, 'The system execution module failed to load: '
+                "only available on Linux systems whith shutdown command.")
```

Comment:
There is a typo here. Could you also change this so it uses either single-quotes or double-quotes throughout? We prefer single-quotes.